.gitignore

```diff
@@ -3,4 +3,5 @@
 recipes/build-info/files/.build_info
 **/debian-packages.list
 rugpi-bakery.toml
-*.log
\ No newline at end of file
+*.log
+core
```

Comment:
This seems to be unrelated, so please remove these changes

Reply:
Yes, but maybe you could include it in the next release.